pkg/kubelet/kubelet.go

```diff
@@ -906,7 +906,13 @@ func (kl *Kubelet) initializeModules() error {
 	}
 
 	// Step 5: Start container manager.
-	if err := kl.containerManager.Start(); err != nil {
+	node, err := kl.getNodeAnyWay()
+	if err != nil {
+		glog.Errorf("Cannot get Node info: %v", err)
+		return fmt.Errorf("Kubelet failed to get node info.")
+	}
+
+	if err := kl.containerManager.Start(node); err != nil {
```

Comment:
ah, so yeah, i see where this is used.... still would prefer we fix kubelet up to have a more targeted interface that we can pass around for kubelet capacity introspection.